lib/main.dart

```diff
@@ -120,6 +129,9 @@ class _HomePageState extends State<HomePage> {
   @override
   Widget build(BuildContext context) {
 
+    Locale local = DemoLocalizations.of(context).locale;
```

Comment:
For this issue, it is better to have a button or something like that so that the user could change the language manually